src/routes/event-log/validation.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+import { IdAndSchemaVersionSchema, InsertionTimestampsSchema } from '../../utils/db.ts';
+
+export const LogItemSource = {
```

Comment:
I'm assuming "special" has to do with possibly making inserts in history prior to the inception of the community hub?(Founds, past directors, etc)